src/Hosting/Hosting/src/GenericHost/GenericWebHostBuilder.cs

```diff
@@ -267,7 +271,7 @@ void ConfigureStartup(HostBuilderContext context, IServiceCollection services)
         return this;
     }
 
-    [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2006:UnrecognizedReflectionPattern", Justification = "We need to call a generic method on IHostBuilder.")]
+    [UnconditionalSuppressMessage("AOT", "IL3050", Justification = "Constructing untrimmed generic types and methods with reference types")]
```

Comment:
Is this still needed - since it seems you moved all of the MakeGeneric... into a helper.